peanut_gb.h

```diff
@@ -1392,7 +1628,11 @@ void __gb_draw_line(struct gb_s *gb)
 	}
 
 	/* If background is enabled, draw it. */
+#if PEANUT_FULL_GBC_SUPPORT
+	if(gb->cgb.cgbMode || gb->hram_io[IO_LCDC] & LCDC_BG_ENABLE)
+#else
 	if(gb->hram_io[IO_LCDC] & LCDC_BG_ENABLE)
+#endif
```

Comment:
If CGB mode is activated, then the LCD background is always drawn? I think this is correct, but just checking.

Reply:
Yes, at the end it then decides whether to show OBJ in front of BG or not.